apps/examples/performance/tls_seclink/tls_client.c

```diff
@@ -0,0 +1,370 @@
+
+static unsigned char rootca[] = ROOT_CA;
+static int rootca_len = sizeof(rootca);
+
+static const unsigned char TLS_DATA[] = "This is from CLIENT";
+
+#define TEST_CASE_COUNT 7
+static unsigned char test_data[4100] = { 0, };
+static int test_size[TEST_CASE_COUNT] = { 64, 128, 256, 512, 1024, 2048, 4096 };
+static uint32_t test_log[2][TEST_CASE_COUNT] = { 0, };
+static uint32_t log_diff = 0;
+static int log_count[2] = { 0, };
+
+#define ENCRYPT 0
+#define DECRYPT 1
+
+#if defined(CONFIG_TLS_SECLINK_USE_FRT_TIMER)
+static struct timer_status_s before;
+static struct timer_status_s after;
+int frt_fd;
+
+#define TLS_START_TIMER(x) \
+	frt_fd = open(CONFIG_TLS_SECLINK_FRT_TIMER_PATH, O_RDONLY); \
+	ioctl(frt_fd, TCIOC_SETMODE, MODE_FREERUN); \
+	ioctl(frt_fd, TCIOC_START, 0); \
+	ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)&before);
+
+#define TLS_STOP_TIMER(x) \
+	ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)&after); \
+	ioctl(frt_fd, TCIOC_STOP, 0); \
+	close(frt_fd); \
+	do { \
+		log_diff = (unsigned long)after.timeleft - (unsigned long)before.timeleft; \
+		test_log[x][log_count[x]++] = log_diff; \
+	} while(0);
+#else
+static struct timespec before;
+static struct timespec after;
+
+#define TLS_START_TIMER(x) clock_gettime(CLOCK_REALTIME, &before);
+
+#define TLS_STOP_TIMER(x) \
+	clock_gettime(CLOCK_REALTIME, &after); \
+	do { \
+		if (after.tv_nsec < before.tv_nsec) { \
+			log_diff = (1000000000UL + (unsigned long)after.tv_nsec) - (unsigned long)before.tv_nsec; \
+		} else {\
+			log_diff = (unsigned long)after.tv_nsec - (unsigned long)before.tv_nsec; \
+		} \
+		test_log[x][log_count[x]++] = log_diff; \
+	} while(0);
+	
+#endif	
+
+static int ssl_preset_suiteb_ciphersuites[] = {
+	MBEDTLS_TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
+	0
+};
+
+static void my_tls_debug(void *ctx, int level, const char *file, int line, const char *str)
+{
+	((void)ctx);
+	((void)level);
+
+	TLS_SECLINK_CLIENT_LOG("%s:%04d: %s", file, line, str);
+	fflush(stdout);
+}
+
+int tls_print_test_log()
+{
+	int i;
+	int j;
+
+	printf("-------------------------------------------------------------------------------------------\n");
+	printf("        | ");
+	for (i = 0; i < TEST_CASE_COUNT; i++) {
+		printf("%9d | ", test_size[i]);
+	}
+	printf("\n");
+	printf("-------------------------------------------------------------------------------------------\n");
+
+	for (i = 0; i < 2;i ++) {
+		printf("%s | ", (i == 0 ? "Encrypt" : "Decrypt"));
+		for (j = 0; j < TEST_CASE_COUNT; j++) {
+			printf("%9lu | ", test_log[i][j]);
+		}
+		printf("\n");
+	}
+	printf("-------------------------------------------------------------------------------------------\n");
+	return 0;
+}
+
+/****************************************************************************
+ * tls_client_seclink
+ ****************************************************************************/
+void *tls_client_seclink_main(void *arg)
+{
+	printf("Hello, TLS Clinet Test\n");
+
+	int ret = 1, len;
```

Comment:
`ret` will be always set from the return of some function. So I think we don't need to initialize.